tasks/report.ts

```diff
@@ -1,37 +1,28 @@
 import { log, reportUnredeemed, wait } from "./lib";
 import yargs from "yargs/yargs";
+import argv from "../src/getClargs";
```

Comment:
any reason for switching to this? with the old way, we specify and enforce the required/optional params per script; with this way, we lose that I think

Reply:
okay I guess we want the single main.ts entry point (and the require chainid is enforced there); could we still use yargs in main for the full enforcement (i.e., no unspecifified optional params per script, keep descriptions/defaults, etc)?

Reply:
You mean make the argv set from getClargs to main?

Reply:
Any way we can have the required args be specific to their script (instead of the full list applying to all). So that, e.g., I get feedback when I leave out a required arg for the script I'm running. Not vital, just a thought

Reply:
Do you mean like this: https://github.com/OffchainLabs/retryables_tracker/blob/add-docker/tasks/report.ts#L4 ?

Reply:
I'm imagining if script A requires argument x, it will throw an error if x is not provided, but if script B doesn't require x, it won't throw an error; I believe as is it doesn't have this property. We can iterate on this later, I don't think it's worth blocking this PR for it 👍